tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -285,7 +285,32 @@ describe('Boundary Timer Event', () => {
 
   });
 
-  it('it cannot attach an incoming sequence flow on Boundary Timer Events');
+  it('it cannot attach an incoming sequence flow on Boundary Events', function() {
+    const taskForBoundaryEventsPosition = { x: 300, y: 200 };
+    dragFromSourceToDest(nodeTypes.task, taskForBoundaryEventsPosition);
+
+    const taskPosition = { x: 400, y: 300 };
+    dragFromSourceToDest(nodeTypes.task, taskPosition);
+
+    const boundaryEvents = [
+      nodeTypes.boundaryEvent,
+    ];
+
+    boundaryEvents.forEach((event) => {
+      const eventPosition = { x: 300, y: 210 };
+      dragFromSourceToDest(event, eventPosition);
+
+      connectNodesWithFlow('sequence-flow-button', taskPosition, eventPosition);
+
+      getElementAtPosition(taskPosition).then(getLinksConnectedToElement).should($links => {
+        expect($links.length).to.eq(0);
```

Comment:
This can be updated to produce a more readable assertion:

```suggestion
        expect($links).to.have.lengthOf(0);
```